kernel/x86_64/KERNEL.SKYLAKEX

```diff
@@ -10,7 +10,7 @@ SGEMMOTCOPY    =  ../generic/gemm_tcopy_4.c
 #DGEMMKERNEL    =  dgemm_kernel_4x8_skylakex.c
 
 DGEMMINCOPY    =  dgemm_ncopy_8_skylakex.c
-DGEMMITCOPY    =  dgemm_tcopy_8_skylakex.c
+#DGEMMITCOPY    =  dgemm_tcopy_8_skylakex.c
```

Comment:
I'm not familiar with the code base. Can you explain what `gemm_tcopy_8` does and how it's different from `gemm_tcopy_4`?

Reply:
that one takes four elements of input at a time and other eight. 4x8 would mean taking that big rectangular block from input.

Reply:
Doesn't that mean `DGEMMINCOPY` and `DGEMMITCOPY` both need to be `gemm_tcopy_4` variants instead of `gemm_tcopy_8` variants?

Reply:
nope.